src/features/tracing/nrfml.ts

```diff
@@ -238,6 +240,11 @@ export const startTrace =
             }
         }
 
+        if (getRefreshOnStart(state) && !formats.includes('live')) {
+            logger.info(`Refreshing values in 5 seconds`);
```

Comment:
I suggest we change values to dashboard, in order to promote consistency.

```suggestion
            logger.info(`Refreshing dashboard in 5 seconds`);
```